qiskit/visualization/utils.py

```diff
@@ -281,40 +281,52 @@ def get_wire_label(drawer, register, index, layout=None, cregbundle=True):
     return wire_label
 
 
-def get_condition_label_val(condition, circuit, cregbundle, reverse_bits):
+def get_condition_label_val(condition, circuit, cregbundle):
```

Comment:
Maybe beyond the scope of this PR, but I think as this utils file keeps growing it would be good at some point to separate these helper functions out into multiple files, in their own utils folder

Reply:
That would be great! And it would be great to get an architecture diagram put together to go with with that, I feel like the visualisation code can be quite confusing for newcomers so something like that would help a lot

Reply:
again, beyond the scope of this PR but food for thought in future 😄